doc/source/administrator/security.md

```diff
@@ -50,6 +50,23 @@ changes to your `config.yaml` file:
 2. Apply the config changes by running `helm upgrade ...`
 3. Wait for about a minute, now your hub should be HTTPS enabled!
 
+***
+**NOTE:**
+
+If the proxy service is of type `LoadBalancer`, then a specific static IP address can be requested (if available) instead of a dynamically acquired one.  
```

Comment:
```suggestion
If the proxy service is of type `LoadBalancer`, which it is by default, then a specific static IP address can be requested (if available) instead of a dynamically acquired one.  
```